include/tvm/runtime/packed_func.h

```diff
@@ -675,12 +687,20 @@ inline TVMType String2TVMType(std::string s) {
     scan = s.c_str();
     LOG(FATAL) << "unknown type " << s;
   }
-  unsigned bits, lanes;
-  char* xdelim;  // "%ux%u"
+  unsigned bits = t.bits, lanes = t.lanes;
+#ifndef _LIBCPP_SGX_CONFIG
+  sscanf(scan, "%ux%u", &bits, &lanes);
+  t.bits = static_cast<uint8_t>(bits);
+  t.lanes = static_cast<uint16_t>(lanes);
+#else
+  char* xdelim;
   bits = strtoul(scan, &xdelim, 10);
-  lanes = strtoul(xdelim + 1, nullptr, 10);
   if (bits != 0) t.bits = static_cast<uint8_t>(bits);
-  if (lanes != 0) t.lanes = static_cast<uint8_t>(lanes);
+  if (scan[xdelim] == 'x') {
```

Comment:
xdelim doesn't return the location of the index, but returns the pointer. We should do instead

if (*xdelim == 'x') {
}

Reply:
Let us still use your logic to replace the sscanf one, as this one seems to be as efficient and we can test it via CI